traffic_portal/app/src/modules/public/login/login.tpl.html

```diff
@@ -21,12 +21,13 @@
     <div class="col-md-12 col-sm-12 col-xs-12">
         <div class="x_panel">
             <div class="x_title">
-                <h2>Login</h2>
+                <h2 ng-show="!oAuthEnabled">Login</h2>
+                <h2 ng-show="oAuthEnabled">Login With Single Sign On</h2>
```

Comment:
instead of `ng-show`, consider `ng-if` since this should never change for a single pageload.

Reply:
done